src/index.html

```diff
@@ -13,20 +13,29 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="/src/style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form action="#" data-qa="big">
+      <label>
+        <input
+          type="text"
+          class="search-bar search-bar__big"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form action="#" data-qa="small">
+      <label>
+        <input
+          type="text"
+          class="search-bar search-bar__small"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
```

Comment:
The `data-qa="hover"` attribute on the small search form input is not specified in the task requirements and should be removed unless explicitly required.